### PR TITLE
Included credscan suppressions and tsa upload tasks

### PR DESCRIPTION
**Pull request by abhipsaMisra**
- Added suppressions file to suppress false positives for CredScan

- Added the tsa upload task

### CODE REVIEW

File: service/Microsoft.Azure.Devices/test/Microsoft.Azure.Devices.Api.Test/ConnectionString/ServiceClientConnectionStringTests.cs [hunk 1]
```diff
@@ -111,7 +111,7 @@ public void ServiceClient_IotHubConnectionStringBuilder_Test()
             Assert.IsNull(iotHubConnectionStringBuilder.SharedAccessSignature);
             Assert.IsTrue(iotHubConnectionStringBuilder.AuthenticationMethod is ServiceAuthenticationWithSharedAccessPolicyKey);
 
-            connectionString = "HostName=acme.azure-devices.net;CredentialType=SharedAccessSignature;SharedAccessKeyName=AllAccessKey;SharedAccessSignature=SharedAccessSignature sr=dh%3a%2f%2facme.azure-devices.net&sig=poifbMLdBGtCJknubF2FW6FLn5vND5k1IKoeQ%2bONgkE%3d&se=87824124985&skn=AllAccessKey";
+            connectionString = "HostName=acme.azure-devices.net;CredentialType=SharedAccessSignature;SharedAccessKeyName=AllAccessKey;SharedAccessSignature=SharedAccessSignature sr=dh%3a%2f%2facme.azure-devices.net&sig=dGVzdFN0cmluZzU=&se=87824124985&skn=AllAccessKey";
```
Comment (prmathur-microsoft) [inline service/Microsoft.Azure.Devices/test/Microsoft.Azure.Devices.Api.Test/ConnectionString/ServiceClientConnectionStringTests.cs]: Is this valid string?

**Comment by CIPop** [comment]
Working with @leantk to unblock testing for this PR.

**Comment by CIPop** [comment]
@abhipsaMisra PTAL It appears that the Analyzers fail as the LTS TSA project isn't found.

**Comment by CIPop** [comment]
@abhipsaMisra this has been opened for too long. Closing for now. Please submit a new PR.

**Comment by CIPop** [comment]
Reopening - this only required an internal service activated.

**Pull request merged**